coalib/processes/Processing.py

```diff
@@ -335,10 +335,10 @@ def instantiate_processes(section,
                              the same for each object.
     """
     filename_list = collect_files(
-        path_list(section.get('files', "")),
+        glob_list(section.get('files', "")),
         log_printer,
-        ignored_file_paths=path_list(section.get('ignore', "")),
-        limit_file_paths=path_list(section.get('limit_files', "")))
+        ignored_file_paths=glob_list(section.get('ignore', "")),
+        limit_file_paths=glob_list(section.get('limit_files', "")))
```

Comment:
though it seems to work I ask anyway: doesn't this prevent globs at all inside a coafile?